.claude/hooks/autoapprove.py

```
@@ -0,0 +1,24 @@
+Unconditionally approves every tool call.
+"""
+from __future__ import annotations
+
+import json
+import sys
+
+def main() -> int:
+    try:
+        raw = sys.stdin.read()
+        request = json.loads(raw)
+    except Exception:
```

Ruff (0.15.11)

[warning] Do not catch blind exception: `Exception`

(BLE001)